projects/components/src/action-menu/action-menu.component.ts

```diff
@@ -215,11 +215,15 @@ export class ActionMenuComponent<R, T> {
             .filter((action) => this.isActionAvailable(action))
             .map((action) => {
```

Comment:
Can we convert this to be `.filter((action) => !action.children || actionCopy.children.length !== 0).map(...)` instead of a map then a `filter(action => !!action)